.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "es5",
+  "tabWidth": 4,
+  "semi": false,
```

Comment:
Do we need all this in our prettierrc.json? Or with the `singleQuote` is enough? Could you check please, we have tabWidht in our prettier extension for VSCode.

Reply:
Its not needed , this is basic config based, recommended on prettier.io . 
Check result:
.tabWidth in the extension (VSCode) is (in my personal case) set to 2 and semi to true. If your experience confirms a shorter config as sufficient lets do this, i was just working based on the documentation, cause i have no further experience to build on.

Reply:
Yes, those things are made for people that don't have prettier at all, we don't need in our case I think. Now you can check what happens if you delete it and if it is all good leave it like that. We want to use as less things we don't need as possible. Thanks for checking

Reply:
seem to work all fine